### DISCUSSION TITLE
Installed 1.1.7 and only 288 of 550 plots are showing in the wallet

### DISCUSSION THREAD

**Discussion by TheChiaLord** [Chia-Network/chia-blockchain#6618] [Support]
Installed 1.1.7 and only 288 of 550 plots are now showing in the wallet/farming. How can I fix this ?

**Comment by loppefaaret** [comment]
Try and remove plot directories, and readd them back into the app

**Comment by BrandtH22** [comment]
@TheChiaLord , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
